components/globals/Alert/Alert.tsx

```diff
@@ -0,0 +1,310 @@
+import { CircleCheckIcon, InfoIcon, WarningIcon } from "@components/form-builder/icons";
+import React, { ReactNode, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+export enum ErrorStatus {
+  SUCCESS = "SUCCESS",
+  WARNING = "WARNING",
+  ERROR = "ERROR",
+  INFO = "INFO",
+}
+
+export const Title = ({
+  children,
+  level = "h2",
+  status,
+}: {
+  children: string;
+  level?: "h2" | "h3" | "h4";
```

Comment:
Should be able to do what we do in ErrorPanel here where we alias to an uppercase prop

```
 headingTag: HeadingTag = "h2",
 ```
 
 and than we can use 
 
 ```
  <HeadingTag className="border-none mb-4">{title || defaultTitle}</HeadingTag>
 ```
 
 vs the current
 
 ```
  const Wrapper = `${level}` as keyof JSX.IntrinsicElements;
 ```

Reply:
that's cool